topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.md

```diff
@@ -0,0 +1,940 @@
+>    > > 2. Selected features are stored in `var: 'selected'`. 
+>    > >
+>    > {: .solution}
+>    >
+>    {: .question}
+>
+{: .hands_on}
+
+
+## Doublet removal
+
+Doublets are removed by calling a customized [**scrublet**](https://github.com/swolock/scrublet) algorithm. *pp.scrublet* will identify potential doublets and the function *pp.filter_doublets* removes them. 
+> <hands-on-title> Scrublet </hands-on-title>
+>
+> 1. {% tool [SnapATAC2 Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/snapatac2_preprocessing/snapatac2_preprocessing/2.5.3+galaxy1) %} with the following parameters:
+>    - *"Method used for preprocessing"*: `Compute probability of being a doublet using the scrublet algorithm, using 'pp.scrublet'`
+>        - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC select_features` (output of **pp.select_features** {% icon tool %})
+>
+> 2. {% tool [SnapATAC2 Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/snapatac2_preprocessing/snapatac2_preprocessing/2.5.3+galaxy1) %} with the following parameters:
+>    - *"Method used for preprocessing"*: `Remove doublets according to the doublet probability or doublet score, using 'pp.filter_doublets'`
+>        - {% icon param-file %} *"Annotated data matrix"*: `Anndata scrublet` (output of **pp.scrublet** {% icon tool %})
+> 3. Rename the generated file to `Anndata 5k PBMC filter_doublets` or add the tag {% icon galaxy-tags %} `filter_doublets` to the dataset
+> 4. {% icon galaxy-eye %} Inspect the general information of the `.h5ad` output
+>
+>    > <question-title></question-title>
+>    >
+>    > ```
+>    > AnnData object with n_obs × n_vars = 4430 × 6062095
+>    >  obs: 'n_fragment', 'frac_dup', 'frac_mito', 'tsse', 'doublet_probability', 'doublet_score'
+>    >  var: 'count', 'selected'
+>    >  uns: 'doublet_rate', 'reference_sequences', 'scrublet_sim_doublet_score'
+>    >  obsm: 'fragment_paired'
+>    > ```
+>    >
+>    > 1. What was removed by **pp.filter_doublets**?
+>    > 2. Where are the new annotations stored?
+>    >
+>    > > <solution-title></solution-title>
+>    > >
+>    > > 1. Cell doublets were removed. `n_obs` was reduced from 4564 to 4430 cells. 
+>    > > 2. The outputs of **pp.scrublet** are stored in observations `obs: 'doublet_probability', 'doublet_score'` and in unstructured annotations `uns: 'scrublet_sim_doublet_score'`. The output of **pp.filter_doublets** is stored in `uns: 'doublet_rate'`. 
+>    > >
+>    > {: .solution}
+>    >
+>    {: .question}
+>
+>
+{: .hands_on}
+
+
+# Dimension reduction
+
+Dimension reduction (also known as embedding) is a very important step during the analysis of single cell data. During this, the complex multi-dimensional data is projected into lower-dimensional space, while retaining as much information as possible. Dimension reduction enables quicker downstream analysis since the data is more simplified and thus the memory usage is reduced. 
```

Comment:
I'm not quite sure about the correct definition of embedding. I've often found phrases such as "the low-dimensional embedding of higher dimensional data". Maybe we can change it to this?
```suggestion
>
```Dimension reduction is a very important step during the analysis of single cell data. During this, the complex multi-dimensional data is projected into lower-dimensional space, while the lower-dimensional embedding of the complex data retains as much information as possible. Dimension reduction enables easier visualization and quicker downstream analysis since the data is more simplified and the memory usage is reduced.